geometry_msgs/Quaternion[] quaternions
